bmds_server/analysis/executor.py

```diff
@@ -26,6 +26,13 @@ def build_frequentist_session(dataset, inputs, options, dataset_options) -> Opti
         (PriorEnum.frequentist_restricted, remap_exponential(restricted_models)),
```

Comment:
can you also write a test for this please in test_executor?

To check these tests locally, you can run this command

```bash
py.test tests/analysis/test_executor.py
```